tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -5698,23 +5742,33 @@ Status ConvertCombinedNMS(OpConverterParams* params) {
 
   if (params->validation_only) return Status::OK();
 
-  // TF op CombinedNonMaxSuppression doesn't have the option of
-  // not normalizing coordinates.
+  // TRT op is_normalized=False treats input corrdinates as pixels and
+  // calculates width/height as (max - min + 1).
+  //
+  // TF op CombinedNonMaxSuppression doesn't care about the normalization and
+  // calculates width/height  as (max-min).
+  //
+  // We set is_normalized = true to be consistent with TF IOU calculaton.
   const bool is_normalized = true;
+
   // Set plugin fields and the field collection
```

Comment:
This comment line here doesn't look very helpful to me now. We can either move it closer to "fields" or remove it.

Reply:
I have removed it.